test/screens_web/views/v2/audio/departures_view_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule ScreensWeb.V2.Audio.DeparturesViewTest do
+  use ScreensWeb.ConnCase, async: true
+
+  alias ScreensWeb.V2.Audio.DeparturesView
+
+  describe "no sections" do
+    setup :empty
```

Comment:
This feels like slightly pre-emptive refactoring given we only use each of the values once. I think understanding these tests would be easier if, within each one, we defined the value and then did our assertion — no [mystery guests](http://xunitpatterns.com/Obscure%20Test.html#Mystery%20Guest) coming in from elsewhere that must be cross-referenced.

```elixir
assigns = %{
  sections: [
    %{type: :normal_section, departure_groups: []},
    %{type: :normal_section, departure_groups: []},
    %{type: :normal_section, departure_groups: []}
  ]
}

assert render(assigns: assigns) =~ "There are no upcoming trips at this time"

Reply:
e1542793